test/tests/fuel-cycle/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  issues = '#82'
+  design = 'ParsedODEKernel.md'
```

Comment:
Please add ODETimeDerivative's documentation here too, since it's an important component.